### ISSUE TITLE
Desktop app: Session object was destroyed

### ISSUE THREAD

**Issue by qris** [OmniDB/OmniDB#792]
I think I left the OmniDB window open for 24 hours, and when I tried to change settings after that, I got a dialog box saying something "Session object was destroyed, please click here to login", which took me to this page:

![image](https://user-images.githubusercontent.com/754175/49860752-58e33b80-fdf2-11e8-8376-d97ae2800333.png)

As this is a desktop app, I have no credentials to login with.

**Comment by qris** [comment]
Yes it did, but would prefer not to have to reopen the app every 24 hours.

**Comment by rafaelthca** [comment]
@qris, that makes sense, will try to simulate this and propose a fix.

Thanks.

**Comment by wind39** [comment]
Hi @rafaelthca ,

Does it have to do with cookie expiration time?

Best regards,

William
